webrtc/RTCDtlsTransport-getRemoteCertificates.html

```diff
@@ -4,104 +4,97 @@
 <script src="/resources/testharness.js"></script>
 <script src="/resources/testharnessreport.js"></script>
 <script src="RTCPeerConnection-helper.js"></script>
+<script src="RTCTransport-helper.js"></script>
 <script>
   'use strict';
 
   // The following helper functions are called from RTCPeerConnection-helper.js:
-  //   exchangeIceCandidates
-  //   doSignalingHandshake
-
-  /*
-    5.5.  RTCDtlsTransport Interface
-      interface RTCDtlsTransport : EventTarget {
-        readonly attribute RTCDtlsTransportState state;
-        sequence<ArrayBuffer> getRemoteCertificates();
-                 attribute EventHandler          onstatechange;
-                 attribute EventHandler          onerror;
-        ...
-      };
-
-      enum RTCDtlsTransportState {
-        "new",
-        "connecting",
-        "connected",
-        "closed",
-        "failed"
-      };
-
-      getRemoteCertificates
-        Returns the certificate chain in use by the remote side, with each certificate
-        encoded in binary Distinguished Encoding Rules (DER) [X690].
-        getRemoteCertificates() will return an empty list prior to selection of the
-        remote certificate, which will be completed by the time RTCDtlsTransportState
-        transitions to "connected".
-   */
-  async_test(t => {
+  //  exchangeIceCandidates
+  //  doSignalingHandshake
+
+  // The following helper functions are called from RTCTransport-helper.js:
+  //  waitForConnectedState
+  //  getDtlsTransportFromSctpTransport
+  //  getDtlsTransportsFromSenderReceiver
```

Comment:
Do we need those comments?
I am fearing that they will be obsolete when/if we start updating the tests and forget to update these comments.

Reply:
Since the helper functions are exposed as global variables, I thought it might be helpful for readers to find out where they come from. If people don't find it useful, we may remove them in new PRs.